doc/setup.md

```diff
@@ -0,0 +1,292 @@
+
+output: <local_sig[i]>
+```
+
+- `block_vss[j, i]` (j = 1, 2, ..., n) are the block VSSs generated by Step 7.
+- `node_private_key[i]` is the private key of Signer[i], generated in Step 2.
+- `aggregated_public_key` is an aggregated public key, generated in Step 4.
+- `node_secret_share[i]` is the secret key share of Signer[i], generated in Step 4.
+- `block` is the genesis block without block proof.
+- `local_sig[i]` is the "local signature" described as γi in the paper [^4]. signer[i] may reveal `local_sig[i]`.
+
+`local_sig[i]` is encoded hex string of ( r | s ).
+   - r - the unsigned big-endian 256-bit encoding of the Schnorr signature's r integer.
+   - s - the unsigned big-endian 256-bit encoding of the Schnorr signature's s integer.
+
+Then each signer broadcasts their local_sig.
+
+## Step 9. Compute the signature of the genesis block.
+
+After collecting `local_sig`, we can compute the signature of the genesis block using `tapyrus-setup computesig`
+
+```
+tapyrus-setup computesig -sig=<local_sig[1]> -sig=<local_sig[2]> ... -sig=<local_sig[n]> -block=<block> -block_vss=1:<block_vss[1, i]> -block_vss=2:<block_vss[2, i]> ... -block_vss=t:<block_vss[n, i]> -node_vss=1:<node_vss[1, i]> -node_vss=2:<node_vss[2, i]> ... -node_vss=n:<node_vss[n, i]>
+
+
+output: <block_with_signature>
+```
+
+- `local_sig[i]` is the local signatures broadcasted by signers.
+- `block_with_signature` is the whole genesis block data with block proof as hex string format.
+
+To make block proof, t `local_sig`s are required.
+`local_sig`s generated by signers are public values, so signers can reveal them as plaintext without any encryption.
+
+## Step 10. genesis.`networkid`
+
+Create file named 'genesis.`networkid`' and fill it with `block_with_signature`.
+
+## Next Step
+
+Now we are ready to start Tapyrus Core and Tapyrus Signer Network.
+
+To start Tapyrus Core Network, see [Start Tapyrus-core nodes](https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/getting_started.md#5-start-tapyrus-core-nodes).
+
+To start Tapyrus Signer Network, go to [How To configure Tapyrus Signer Network](./configuration.md). The secret values or keys refered in this document will be required to start Tapyrus Signer Network.
+
+- `node_private_key[i]` and `node_public_key[i]` in Step 1
+- `aggregated_public_key` and `node_secret_share[i]` in Step 3
+
+In above list, `node_private_key` and `node_secret_share` must be treated as secret.
+
+## Appendix A: Encoding and Encrypting the VSS
+
+In this section, we describe the protocol to encode/encrypt the VSS.
+`node_vss[i, j]` in Step 3 and `block_vss[i, j]` in Step 7 have same structure.
+
+Below in this section, notation `vss` means `node_vss[i, j]` or `block_vss[i, j]`.
+
+### Structure of vss
```

Comment:
I noticed that node_vss and block_vss have different structure. 
The node_vss should not have `positive_*` `negative_*`. It just has `commitments` and `secret`.

I think it should be separate two different struct for each.